### PR TITLE
Productionize properly

### PR DESCRIPTION
**Pull request by robinbb**

### CODE REVIEW

File: ops/productionize [hunk 1]
```diff
@@ -7,6 +7,11 @@ set -o pipefail
 #
 SRC="$(git rev-parse --show-toplevel)"
 
+# Confirm that the binary is present and working.
+#
+BIN="$SRC"/rust/target/release/mina-indexer
+"$BIN" --help
```
Comment (trevorbernard) [inline ops/productionize:13]: Printing the `--version` would be more useful methinks

Comment (robinbb) [reply ops/productionize]: Okay.

**Comment by robinbb** [comment]
Tested on the prod machine!

**Review by trevorbernard — APPROVED** [review]

**Comment by trevorbernard** [comment]
:shipit:

**Pull request merged**